website/versioned_docs/version-5.x/ecosystem-alpinejs.md

```diff
@@ -0,0 +1,236 @@
+                <div class="font-medium text-xl truncate" x-text="user.name"></div>
+                <div class="truncate uppercase text-xs text-gray-500 font-semibold 
+                pb-2 tracking-widest" x-text="user.company.name"></div>
+                <div class="" x-text="user.phone"></div>
+                <a class="text-blue-600 hover:text-blue-700 mr-4 block"     
+                    x-bind:href="'mailto:' + user.email" x-text="user.email"></a>     
+                <a class="text-blue-600 hover:text-blue-700 block" 
+                x-bind:href="'https://' + user.website" x-text="user.website"></a>
+                </div>
+            </div>
+            </div>
+        </template>
+        <!-- end: user card -->
+        </div>
+    </div>
+  `;
+
+const appDataFn = ({ title, name }) => ({
+  title,
+  intro:
+    'Implement a simple <code class="text-md text-pink-600">fetch()</code> request to render a list of items using Alpine.js :)',
+  users: [],
+  open: false,
+  name,
+});
+
+const appXInitFn = (id) => {
+  return fetch('https://jsonplaceholder.typicode.com/users')
+    .then((response) => response.json())
+    .then((data) => (document.querySelector(`#${id}`).__x.$data.users = data));
+};
+
+const opts = {
+  template: appTemplate,
+  xData: (data) => appDataFn(data), // pass props to x-data
+  xInit: appXInitFn
+};
+
+const alpinejslifecycles = singleSpaAlpinejs(opts);
+
+export const bootstrap = alpinejslifecycles.bootstrap;
+export const mount = alpinejslifecycles.mount;
+export const unmount = alpinejslifecycles.unmount;
+```
+
+#### Via cdn
+
+Example usage when installed via CDN:
+
+```js
+const alpinejsApp = window.singleSpaAlpinejs.default(opts);
+
+singleSpa.registerApplication({
+  name: 'name',
+  app: alpinejsApp,
+  activeWhen: () => true,
+});
+```
+
+## API / Options
+
+single-spa-html is called with an object that has the following properties:
+
+- `template` (required): An HTML string or a function that returns a string. The function will be called with the single-spa custom props. The returned string is injected into the DOM during the single-spa mount lifecycle.
+- `domElementGetter` (optional): A function that returns the dom element container into which the HTML will be injected. If omitted,
+  a default implementation is provided that wraps the template in a `<div>` that is appended to `document.body`.
+- `xData` (optional): An object or a function or a function that returns a promise.The returned string is injected into the DOM as the `x-data` attribute during the single-spa mount lifecycle.
+- `xInit` (optional): A function or a function that returns a promise.The function provided is added to the global scope and the function initiation along with the root dom element id as a parameter is injected into the DOM as the `x-init` attribute during the single-spa mount lifecycle.Please note the `xData` attribute _must_ be provided otherwise the `xInit` attribute will be ignored.
```

Comment:
Should we document the name of the global variable where xInit is stored?

```suggestion
- `xInit` (optional): A function or a function that returns a promise. The function provided is added to the global scope and the function initiation along with the root dom element id as a parameter is injected into the DOM as the `x-init` attribute during the single-spa mount lifecycle.Please note the `xData` attribute _must_ be provided otherwise the `xInit` attribute will be ignored.
```

Reply:
Agreed .. will document the name of the global variable for xInit functions